content/docs/reconciliation.md

```diff
@@ -109,11 +108,11 @@ If you implement it naively, inserting an element at the beginning has worse per
 </ul>
 ```
 
-React will mutate every child instead of realizing it can keep the `<li>Duke</li>` and `<li>Villanova</li>` subtrees intact. This inefficiency can be a problem.
+React akan mengubah setiap *child* daripada menyadari itu dapat menjaga keutuhan sub-pohon `<li>Duke</li>` dan `<li>Villanova</li>`. Ketidakefisienan ini bisa menjadi masalah.
 
-### Keys {#keys}
+### Kunci {#keys}
```

Comment:
```suggestion
### Key {#keys}
```

As per the [glossary](https://github.com/reactjs/id.reactjs.org/wiki/Glosarium#k), *key* tidak perlu diterjemahkan.